lbz/configuration.py

```diff
@@ -0,0 +1,60 @@
+from abc import ABCMeta, abstractmethod
+from os import getenv
+from typing import Any, Callable, Generic, Optional, TypeVar
+
+from lbz.aws_ssm import SSM
+from lbz.exceptions import ConfigurationMissingError, ConfigurationParsingError
+
+T = TypeVar("T")
+
+
+class ConfigValue(Generic[T], metaclass=ABCMeta):
+    """Mandatory Configuration
+
+    This class is not supporting None as outcome value.
+    """
+
+    def __init__(
+        self,
+        key: str,
+        parser: Callable[[str], T] = str,  # type: ignore
+        default: T = None,
+    ):
+        self._key = key
+        self._parser = parser
+        self._default = default
+        self._value: Optional[T] = None
+
+    @abstractmethod
+    def getter(self) -> Any:
+        pass
+
+    @property
+    def value(self) -> T:
+        if self._value is None:
+            val = self.getter()
+            if val is not None:
```

Comment:
New `pylint` configuration will point that out anyway:
```suggestion
            if (val := self.getter()) is not None:
```